addons/web_tour/views/tour_views.xml

```diff
@@ -1,5 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+        <record id="edit_tour_form" model="ir.ui.view">
+            <field name="model">web_tour.tour</field>
+            <field name="arch" type="xml">
+                <form string="Menu" editable="bottom">
```

Comment:
editable bottom on a form view? I'm not sure if that is doing anything.

Reply:
@seb-odoo removed the editable bottom